samples/samples/connection_sample.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+
+from google.cloud import spanner
+from google.cloud import spanner_dbapi
+
+from google.cloud.spanner_dbapi import connection, InterfaceError
+
+
+def list_all():
+    spanner_client = spanner.Client()
+
+    for inst in spanner_client.list_instances():
+        print(inst.name)
+        for db in inst.list_databases():
+            print(db.name)
+
+
+def create_connection(instance_id, database_id):
+    my_connection = spanner_dbapi.connect(instance_id, database_id)
+
+    print("Connection is created.")
+
+
+def get_cursor(instance_id, database_id):
+    # Opened connection
+    my_connection = spanner_dbapi.connect(instance_id, database_id)
+    cursor = my_connection.cursor()
+    print("Got cursor.")
+
+    # Closed connection
+    my_connection.close()
+    try:
+        cursor = my_connection.cursor()
+    except InterfaceError:
+        print("Can't got cursor. Connection is closed.")
+
+
+def run_sql_in_snapshot(instance_id, database_id, sql):
+    my_connection = spanner_dbapi.connect(instance_id, database_id)
+
+    for table in my_connection.list_tables():
+        print(table, my_connection.get_table_column_schema(table[0]))
+
+    snapshot = my_connection.read_snapshot()
+
+    result_list = my_connection.run_sql_in_snapshot(sql)
+    print("SQL result:", result_list)
```

Comment:
We're going to remove these methods from `Connection` class, as PEP 249 doesn't describe snapshots, `list_tables()` and `get_table_column_scheme()` methods.

Reply:
So just remove them?

Reply:
Yes. I think some of them are already deleted. Our `db_api` should only include methods described in PEP 249.